content/es/docs/tasks/manage-daemon/_index.md

```diff
@@ -1,4 +1,4 @@
 ---
-title: Gestionar y ejecutar demonios
+title: Gestionar y ejecutar procesos
```

Comment:
Igual que antes, los DaemonSets son tipos muy específicos de procesos, no creo que se pueda generalizar.

https://es.wikipedia.org/wiki/Daemon_(inform%C3%A1tica)
https://wiki.archlinux.org/index.php/Daemons_(Espa%C3%B1ol)

Qué opináis @glo-pena @electrocucaracha?

Reply:
Desde mi punto de vista, daemons (*nix) y servicios (windows) son procesos que se ejecutan en segundo plano por el sistema operativo. DaemonSet es un concepto acuñado por Kubernetes para simular esa ejecucion en segundo plano, en otras palabras no creo que sea buena idea traducir DaemonSet pero si daemons.

Reply:
@raelga @electrocucaracha @glo-pena Voy a plantear este _dilema_ en Slack a ver qué opina la gente...

Reply:
![image](https://user-images.githubusercontent.com/795289/59354803-25905380-8d26-11e9-9d70-d97bbc1861bf.png)

https://kubernetes.slack.com/archives/CH7GB2E3B/p1557842074060400